### DISCUSSION TITLE
Collapsed Version Qualifier

### DISCUSSION THREAD

**Discussion by flyingfishflash** [mooltiverse/nyx#256] [Q&A] [answered]
Hi - 

1.2.3-mybranchname.1

One item I'm having a hard time understanding is how the '.1' is incremented in the collapsed version qialifier?

No matter what I do I'm unable to come up with a configuration where this number is incremented beyond 1. I'm sure I'm doing something wrong.

When I read this: https://levelup.gitconnected.com/nyx-the-semantic-release-automation-tool-4e2dfa949f38
or look at the examples here: https://mooltiverse.github.io/nyx/example/user/git-history-examples/

It indicates to me that in a pre-release (or any branch with collapsed versionining?), each significant commit would cause nyx to increment the collapsed version qualifier.

Is there a minimally viable example of how to configure that behavior?

Given just a simple repo with a single file and a main branch + pre-release or feature style branch?

Thanks
-Greg

**Comment by flyingfishflash** [comment]
In any case I realized my error in understanding was that a tag is required to increment the numeric portion of the collapsed version qualifier. It's not simply a count of significant commits since the prime version appended to the collapsed version qualifier.

**Comment by flelli** [comment, marked as answer]
Glad to see you figured it out @flyingfishflash .

For others to land on this topic, [this page](https://mooltiverse.github.io/nyx/faq/user/concepts/how-does-collapsed-versioning-work/) may also help.
